shoop/themes/classic_gray/templates/classic_gray/shoop/front/errors/400.jinja

```diff
@@ -0,0 +1,17 @@
+{% extends "shoop/front/base.jinja" %}
+
+{% block title %}{% trans %}Bad Request{% endtrans %} (400){% endblock %}
+{% block content_title %}{% trans %}Bad Request{% endtrans %} (400){% endblock %}
```

Comment:
Instead of repeating yourself, you can use [`{{ self.title() }}` etc. to call other blocks from blocks](http://jinja.pocoo.org/docs/dev/templates/#child-template):

> If you want to print a block multiple times, you can, however, use the special self variable and call the block with that name:
```jinja
<title>{% block title %}{% endblock %}</title>
<h1>{{ self.title() }}</h1>
{% block body %}{% endblock %}
```